packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -903,7 +882,7 @@ describeNoCompat("stashed ops", (getTestObjectProvider) => {
 
 			[...Array(lots).keys()].map((i) => dataStore.root.set(`test op #${i}`, i));
 
-			const pendingState = getPendingStateWithoutClose(container);
+			const pendingState = await container.getPendingLocalState?.();
```

Comment:
I don't really understand this change to this test, previously. getPendingStateWithoutClose called closeAndGetPendingLocalState, and now this is calling getPendingLocalState

Reply:
getPendingStateWithoutClose is basically getPendingLocalState, since the method just override the close call to be undefined and avoid closing. That made sense when there was no getPendingLocalState, since previously we only had closeAndGetPendingLocalState.

Reply:
but why didn't close get called previously? the previous code made it such that it would assert if close was called. this test no longer checks or prevents close from being called.

Reply:
i see. it just blocked close with error. i'd prefer if we minimally updated tests.